src/modm/board/feather_m0/board.hpp

```diff
@@ -100,18 +97,18 @@ struct SystemClock
 	}
 };
 
-using LoggerDevice = modm::IODeviceWrapper< Uart0, modm::IOBuffer::BlockIfFull >;
-using Leds = SoftwareGpioPort< Led >;
+using LoggerDevice = modm::IODeviceWrapper<Uart0, modm::IOBuffer::BlockIfFull>;
+using Leds = Led;
 
 inline void
 initialize()
 {
 	SystemClock::enable();
 	SysTickTimer::initialize<SystemClock>();
-	Uart0::connect<Rx::Pad3, Tx::Pad2>();
+	Uart0::connect<Rx, Tx>();
```

Comment:
I'd prefer to have Pin and Signal name in the same line.
```suggestion
	Uart0::connect<Rx::Rx, Tx::Tx>();
```

Reply:
But they are totally different, for example you can use the `Rx` signal from the `Tx` pin (`Tx::Rx`) if we make that decision, which makes no sense to me.

The problem is that the signal to pin mapping is much more flexible on SAM, so we need to be a bit careful keeping them apart.